Example.md

```diff
@@ -0,0 +1,977 @@
+      type: git
+    - name: runtime-image
+      type: image
+  tasks:
+    - name: build-image
+      taskRef:
+        name: buildah-task
+      resources:
+        inputs:
+          - name: source
+            resource: source-repo
+        outputs:
+          - name: image
+            resource: runtime-image
+    - name: deploy-image
+      runAfter:
+        - build-image
+      taskRef:
+        name: deploy-using-kubectl-task
+      resources:
+        inputs:
+          - name: source
+            resource: source-repo
+          - name: image
+            resource: runtime-image
+      params:
+      - name: PATHTODEPLOYMENT
+        value: ${DEPLOYMENT_PATH}
+      - name: YAMLPATHTOIMAGE
+        value: "spec.template.spec.containers[0].image"
+      - name: NAMESPACE
+        value: ${USER}-dev-environment
+EOF
+```
+### stage-cd-pipeline
+
+```shell
+cat <<EOF | oc apply -f -
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: stage-cd-pipeline
+spec:
+  resources:
+    - name: source-repo
+      type: git
+  tasks:
+    - name: apply-source
+      taskRef:
+        name: deploy-from-source-task
+      resources:
+        inputs:
+          - name: source
+            resource: source-repo
+      params:
+      - name: NAMESPACE
+        value: ${USER}-stage-environment
+EOF
+```
+## Create EventListener
+
+### cicd-event-listener
+
+In this EventListener, the following triggers are configured.
+
+* When Pull Request is Opened/Synchronized on repo `taxi`, it triggers [dev-ci-build-from-pr-binding](#dev-ci-build-from-pr-binding)/[dev-ci-build-from-pr-template](#dev-ci-build-from-pr-template)
+* When Pull Request is Pushed to master branch of repo `taxi`, it triggers [dev-cd-deploy-from-master-binding](#dev-cd-deploy-from-master-binding)/[dev-cd-deploy-from-master-template](#dev-cd-deploy-from-master-template)
+* When Pull Request is Opened/Synchronized on repo `taxi-stage-config`, it triggers [stage-ci-dryrun-from-pr-binding](#stage-ci-dryrun-from-pr-binding)/[stage-ci-dryrun-from-pr-template](#stage-ci-dryrun-from-pr-template)
+* When Pull Request is Pushed to master branch of repo `taxi-stage-config`, it triggers [stage-cd-deploy-from-push-binding](#stage-cd-deploy-from-push-binding)/[stage-cd-deploy-from-push-template](#stage-cd-deploy-from-push-template)
+
+For more information about GitHub REST API, please see https://developer.github.com/v3/activity/events/types/#pushevent
+
+Follow the these steps to create `cicd-event-listener`
+
+
+* Save the following content to `temp.yaml`
+
+```shell
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: cicd-event-listener
+spec:
+  serviceAccountName: demo-sa
+  triggers:
+    - name: dev-ci-build-from-pr
+      interceptors:
+        - cel:
+            filter: (header.match("X-GitHub-Event", "pull_request") && body.action == "opened" || body.action == "synchronize") && body.pull_request.head.repo.full_name == "$GITHUB_USER/taxi"
```

Comment:
I've updated the event-listener YAML file, you'll want to replace `"` with `'`.

Reply:
If this works (which it does), then I would rather leave it.